lib/databasedotcom/flow.rb

```diff
@@ -0,0 +1,25 @@
+module Databasedotcom
+  # A Salesforce API endpoint that can trigger predefinted flows in salesforce to kick off salesforce side processes.
+  # A flow can be triggered by its unique developer name and passed a hash of input variables.
```

Comment:
`# A flow can be triggered by its unique name and can be passed a hash of input variables.`